src/tools/dartpad/index.md

```diff
@@ -3,13 +3,12 @@ title: DartPad
 description: The tool that lets you interactively play with Dart in a browser.
 ---
 
-DartPad is an open-source tool that
-lets you play with the Dart language in any modern browser.
+DartPad is an [open source](https://github.com/dart-lang/dart-pad) tool
+that lets you play with the Dart language in any modern browser.
 Many pages in this site — especially [codelabs](/codelabs) —
 have [embedded DartPads](#embedding).
-To get a DartPad as big as your browser window, go to the
-<a href="{{site.dartpad}}" target="_blank" rel="noopener">DartPad
```

Comment:
@kwalrath is the `rel="noopener"` required? IIUC, [noopener](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener) is a security defense against untrusted content.

Reply:
It was a mistake for me to completely remove the `target="_blank"` as that allows the link to open in a new tab/window. I've added it back with Kramdown syntax to maintain that behavior.

According to the link you mentioned, this also "implicitly provides the same rel behavior as setting rel="noopener"" so we should be good.